<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>817</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BugzBunny</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>0</y>
      <width>241</width>
      <height>561</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="profileButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Helvetica</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Мой профиль</string>
     </property>
    </widget>
    <widget class="QPushButton" name="commentButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Helvetica</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Мои ответы</string>
     </property>
    </widget>
    <widget class="QPushButton" name="taskButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Helvetica</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Мои задачи</string>
     </property>
    </widget>
    <widget class="QPushButton" name="projectButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Helvetica</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Мои проекты</string>
     </property>
    </widget>
    <widget class="QPushButton" name="grpButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Helvetica</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Мои группы</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>601</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="rightButton">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>10</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QLabel" name="title">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>401</width>
       <height>31</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>601</width>
      <height>511</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>31</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Имя:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>70</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Bunny</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>40</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Bugz</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>60</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Фамилия:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Дата рождения:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>130</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>bugzbunny@ny.com</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>230</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Изменить пароль:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>120</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>E-mail:</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdit">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>100</y>
       <width>110</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="saveProfile">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>170</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Сохранить изменения</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>240</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>старый пароль</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_5">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>270</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>новый пароль</string>
     </property>
    </widget>
    <widget class="QPushButton" name="savePass">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>300</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Сменить пароль</string>
     </property>
    </widget>
    <zorder>label</zorder>
    <zorder>lineEdit_2</zorder>
    <zorder>lineEdit</zorder>
    <zorder>label_2</zorder>
    <zorder>label_3</zorder>
    <zorder>label_5</zorder>
    <zorder>label_4</zorder>
    <zorder>dateEdit</zorder>
    <zorder>saveProfile</zorder>
    <zorder>lineEdit_4</zorder>
    <zorder>lineEdit_5</zorder>
    <zorder>savePass</zorder>
    <zorder>lineEdit_3</zorder>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>49</y>
      <width>601</width>
      <height>511</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTableWidget" name="projectView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>601</width>
       <height>461</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="myProjects">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Мои проекты</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="allProjects">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Все проекты</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTableWidget" name="projectView_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>601</width>
       <height>461</height>
      </rect>
     </property>
    </widget>
    <zorder>myProjects</zorder>
    <zorder>allProjects</zorder>
    <zorder>projectView</zorder>
    <zorder>projectView_2</zorder>
   </widget>
   <zorder>frame_3</zorder>
   <zorder>frame</zorder>
   <zorder>frame_2</zorder>
   <zorder>frame_4</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>817</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Треккер</string>
    </property>
    <addaction name="action_15"/>
    <addaction name="action"/>
    <addaction name="action_11"/>
    <addaction name="action_12"/>
    <addaction name="action_13"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Проекты</string>
    </property>
    <addaction name="action_9"/>
    <addaction name="action_10"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Задачи</string>
    </property>
    <addaction name="action_6"/>
    <addaction name="action_7"/>
    <addaction name="action_8"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Профиль</string>
    </property>
    <addaction name="action_16"/>
    <addaction name="separator"/>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
    <addaction name="separator"/>
    <addaction name="action_4"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>Группы</string>
    </property>
    <addaction name="action_5"/>
    <addaction name="action_14"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_5"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action">
   <property name="text">
    <string>Статистика работы</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Настройки аккаунта</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Статистика</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>Создать задачу</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>Назначенные мне</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>Созданные мной</string>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>Создать проект</string>
   </property>
  </action>
  <action name="action_10">
   <property name="text">
    <string>Открыть проект</string>
   </property>
  </action>
  <action name="action_11">
   <property name="text">
    <string>Пользователи</string>
   </property>
  </action>
  <action name="action_12">
   <property name="text">
    <string>Проекты</string>
   </property>
  </action>
  <action name="action_13">
   <property name="text">
    <string>Задачи</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>Создать группу</string>
   </property>
  </action>
  <action name="action_14">
   <property name="text">
    <string>Мои группы</string>
   </property>
  </action>
  <action name="action_15">
   <property name="text">
    <string>Войти</string>
   </property>
  </action>
  <action name="action_16">
   <property name="text">
    <string>Войти</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
